app/models/item.rb

```diff
@@ -1,3 +1,5 @@
 class Item < ActiveRecord::Base
-  validates :title, :description, :price, presence: true
+  validates  :description, :price, presence: true
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.